docs/setup/install/brew.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[brew]]
+=== Install {kib} on macOS with Homebrew
+
+Elastic publishes Homebrew formulae so you can install {kib} with the https://brew.sh/[Homebrew] package manager.
+
+To install with Homebrew, you first need to tap the Elastic Homebrew repository:
+
+[source,sh]
+-------------------------
+brew tap elastic/elastic-stack
+-------------------------
+
+Once you've tapped the Elastic Homebrew repo, you can use `brew install` to
+install {kib}:
+
+[source,sh]
+-------------------------
+brew install elastic/elastic-stack/kibana
+-------------------------
+
+This installs the most recently released default distribution of {kib}.
+Append the version number to install a previous version, for example
+`elastic/elastic-stack/kibana@6.8`. To install the OSS distribution,
+specify `elastic/elastic-stack/kibana-oss`.
+
+[[brew-layout]]
+==== Directory layout for Homebrew installs
+
+When you install {kib} with `brew install`, the config files, logs,
+and data directory are stored in the following locations.
+
+[cols="<h,<,<m,<m",options="header",]
+|=======================================================================
+| Type | Description | Default Location | Setting
+| home
+  | Kibana home directory or `$KIBANA_HOME`
+  | /usr/local/var/homebrew/linked/kibana
+ d|
+
+| bin
+  | Binary scripts including `kibana` to start a node
+    and `kibana-plugin` to install plugins
+  | /usr/local/var/homebrew/linked/kibana/bin
+ d|
+
+| conf
+  | Configuration files including `kibana.yml`
+  | /usr/local/etc/kibana
```

Comment:
The table isn't formatting correctly.  You need to add a blank column to this entry.